.clang-tidy

```diff
@@ -29,19 +39,24 @@ hicpp-avoid-goto,
 misc-unused-alias-decls,
 misc-unused-using-decls,
 modernize-*,
+-modernize-avoid-c-arrays,
 -modernize-concat-nested-namespaces,
+-modernize-macro-to-enum,
 -modernize-return-braced-init-list,
 -modernize-use-auto,
 -modernize-use-default-member-init,
 -modernize-use-using,
 -modernize-use-trailing-return-type,
 -modernize-use-nodiscard,
+-modernize-redundant-void-arg,
 performance-*,
+-performance-move-const-arg,
```

Comment:
I've never had issues with this check. You may want to tune check options slightly to not recommend moving into const ref, but it's otherwise reasonable.